autogtp/Game.cpp

```diff
@@ -179,8 +180,10 @@ bool Game::gameStart(const VersionTuple &min_version) {
     // check any return values.
     checkVersion(min_version);
     QTextStream(stdout) << "Engine has started." << endl;
-    sendGtpCommand(m_timeSettings);
-    QTextStream(stdout) << "Infinite thinking time set." << endl;
+    for (auto command : m_commands) {
+        sendGtpCommand(command);
+    }
+    QTextStream(stdout) << "Thinking time set." << endl;
```

Comment:
You have to wait for and parse the answer of each command before you send the next.

Reply:
Actually, doesn't `sendGtpCommand()` already do this?
I'll add catching of a it returning false (command failing) and exit though.